core/src/proc_macros_support.rs

```diff
@@ -0,0 +1,23 @@
+use jsonrpsee_types::ErrorObjectOwned;
+
+#[cold]
```

Comment:
So, I suppose these `#[cold] attr` will not inline the given code?
What is the rationale behind it?